db-client/package.json

```diff
@@ -22,12 +22,22 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@binaris/shift-interfaces-node-client": "^0.4.0",
+    "@binaris/shift-interfaces-node-client": "^0.4.0-nudb.1",
```

Comment:
No need for caret, for rush style dependencies.

Reply:
Done!